lib/worker.js

```diff
@@ -214,12 +214,12 @@ class AssetComputeWorker {
         }
 
         // 2. check and log resulting rendition
+        this.timers.callbackProcessing += callbackTimer.end();
         console.log(`worker() callback generated rendition in ${callbackTimer.durationSec()} seconds: ${rendition.name}`);
 
         // 3. post processing for images
-        this.timers.currentPostProcessing = new Timer();
-        const postProcessingTimer = this.timers.currentPostProcessing;
-
+        // start post-processing timer
+        const postProcessingTimer = this.timers.currentPostProcessing = new Timer();
```

Comment:
I actually found it more confusing when they were on separate lines. 

This is because when a variable is defined, I expect to use it. However, we don't expect to use `this.timers.currentPostProcessing` but instead, refer to it as `postProcessingTimer`. When they were defined on separate lines, I kept getting confused about whether to use `this.timers.currentPostProcessing` or `postProcessingTimer`

Reply:
Some people may read this different ways, let me explain: 
1. `const postProcessingTimer = this.timers.currentPostProcessing = new Timer();` can be understood as: create a new timer and assign it to `this.timers.currentPostProcessing`. Then, assign `this.timers.currentPostProcessing` to `postProcessingTimer`. This is also what the code will do here.
2. `const postProcessingTimer = this.timers.currentPostProcessing = new Timer();` can be understood as: try to instantiate a new Timer() and assign the object's reference to `this.timers.currentPostProcessing`. Then, if the assignment worker, assign 0 (true) to `postProcessingTimer`, or any other value otherwise. That is a behavior you will see in C code, as this would be what C would do (Note: this is actually a way in C to check if memory allocation worked or not, it's not some rare thing). Therefore, that line of code may confuse people coming from C/C++ background. (That's a reason why you I always tried to avoid that, because different people with different development background can be confused by that).

Reply:
For now, I'll keep it as is (1 liner) since that's how we define it in other places in the code.